src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/AnyOS/ManagedPal.KeyTrans.cs

```diff
@@ -223,5 +222,48 @@ internal byte[] DecryptCek(X509Certificate2 cert, RSA privateKey, out Exception
             }
 #endif
         }
+
+        private static bool TryGetRsaOaepEncryptionPadding(
+            ReadOnlyMemory<byte>? parameters,
+            out RSAEncryptionPadding rsaEncryptionPadding,
+            out Exception exception)
+        {
+            exception = null;
+            rsaEncryptionPadding = null;
+            if (parameters == null || parameters.Value.IsEmpty)
```

Comment:
Personal style nit:

I like control blocks to have whitespace separation.  So a blank line between the out-parameter initialization and this if, and then a blank line after the close curly of this if.